src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,22 @@
 import React from 'react';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+import './UserInfo.scss';
+
+import { User } from '../../types/User';
+
+type Props = {
+  user: User,
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user;
```

Comment:
If you have < 3 attribues on a tag and the values are short write the tag in one line (to make it easier to write and read)